content-services/latest/install/ansible.md

```diff
@@ -85,73 +84,93 @@ If you're using the Enterprise edition of Content Services, then you need creden
-| Search Services | 2.0.5 | 2.0.4 | 2.0.2 | 2.0.1.1 | 2.0.3 |
-| Search Enterprise | 3.2.0 | 3.1.1 | 3.1.1 | - | - |
-| All-In-One Transform Engine | 3.0.0 | 2.5.7 | 2.5.6 | 2.3.10 | 2.5.7 |
-| AOS | 1.5.0 | 1.4.1 | 1.4.0 | 1.4.0 | - |
-| GoogleDocs | 3.3.0 | 3.2.2 | 3.2.1 | 3.2.1 | - |
-| Digital Workspace | 3.1.0 | 2.9.0 | 2.6.0 | 2.1.0 | N/A |
-| Transform Router | 2.0.0 | 1.5.3 | 1.5.1 |1.3.2 | N/A |
-| Shared File Store | 2.0.0 | 0.16.1 | 0.16.1 | 0.13.0 | N/A |
-| Sync Service | 3.8.0 | 3.6.0 | 3.5.0 | 3.4.0 | N/A |
+| Component | 7.4 Enterprise | 7.3 Enterprise | 7.2 Enterprise | 7.1 Enterprise | 7.0 Enterprise | Community |
+|-|-|-|-|-|-|-|
+| OpenJDK | 17.0.3 | 17.0.3 | 11.0.15 | 11.0.15 | 11.0.15 | 17.0.03 |
+| Apache Tomcat | 9.0.59 | 9.0.59 | 9.0.59 | 9.0.59 | 8.5.76 | 9.0.59 |
+| PostgreSQL | 14.x | 14.x | 13.x | 13.x | 13.x | 14.x |
+| Apache ActiveMQ | 5.16.6 | 5.16.6 | 5.16.6 | 5.16.6 | 5.16.6 | 5.16.6 |
+| Repository | 7.4.0 | 7.3.0 | 7.2.1 | 7.1.1 | 7.0.1.4 | 7.4.0 |
+| Share | 7.4.0 | 7.3.0 | 7.2.1 | 7.1.1 | 7.0.1.4 | 7.4.0 |
+| Search Services | 2.0.6.1 | 2.0.5 | 2.0.4 | 2.0.2 | 2.0.1.1 | 2.0.6.1 |
+| Search Enterprise | 3.3.0 | 3.2.0 | 3.1.1 | 3.1.1 | - | N/A |
+| All-In-One Transform Engine | 3.1.0 | 3.0.0 | 2.5.7 | 2.5.6 | 2.3.10 | 3.1.0 |
+| AOS | 1.6.0 | 1.5.0 | 1.4.1 | 1.4.0 | 1.4.0 | 1.6.0 |
+| GoogleDocs | 3.4.0 | 3.3.0 | 3.2.2 | 3.2.1 | 3.2.1 | 3.4.0 |
+| Digital Workspace | 4.0.0 | 3.1.0 | 2.9.0 | 2.6.0 | 2.1.0 | N/A |
+| Transform Router | 2.1.0 | 2.0.0 | 1.5.3 | 1.5.1 |1.3.2 | N/A |
+| Shared File Store | 2.1.0 | 2.0.0 | 0.16.1 | 0.16.1 | 0.13.0 | N/A |
+| Sync Service | 3.9.0 | 3.8.0 | 3.6.0 | 3.5.0 | 3.4.0 | N/A |
 
 ## Set up Ansible
 
 Not all distributions of Linux may match the version requirements for Ansible and its dependencies. Below we describe how to configure a control node with one of the many ways to set a Python virtual environment. With Python `virtualenvs` you can install the exact same versions of Ansible we use when testing without impacting your system installation of Python. By doing so, you're ensuring the best chances of success.
 
-1. Download the Ansible playbook [zip file](https://nexus.alfresco.com/nexus/service/local/repositories/releases/content/org/alfresco/alfresco-ansible-deployment/2.2.1/alfresco-ansible-deployment-2.2.1.zip){:target="_blank"}.
+1. Download the Ansible playbook [zip file](https://nexus.alfresco.com/nexus/service/local/repositories/releases/content/org/alfresco/alfresco-ansible-deployment/2.3.0/alfresco-ansible-deployment-2.3.0.zip){:target="_blank"}.
 
    If you're not working directly on the control node, transfer the ZIP file to the control node together with the SSH private key required to login to the target machines, and SSH into the control node.
 
    ```bash
    scp  alfresco-ansible-deployment-<version>.zip user@controlnode:
-   scp  ~/.ssh/ansible_rsa user@controlnode:.ssh
+   scp  -i ~/.ssh/ansible_rsa user@controlnode:.ssh

```

Comment:
We have an error in our doc here.
 Please use instead:
```suggestion
  ssh-copy-id -i ~/.ssh/ansible_rsa user@controlnode
```

Reply:
I added the correction to a new commit. Thank you.